lib/firespring_dev_commands/git.rb

```diff
@@ -424,6 +425,22 @@ def clone_repo(dir:, repo_name:, repo_org: 'firespring', branch: nil, depth: nil
       g.fetch('origin', prune: true)
     end
 
+    def commit_status(repository:, branch:, status:, organization: 'firespring', options: {})
+      token = ENV['GITHUB_TOKEN'].to_s.strip
```

Comment:
I haven't been terribly consistent on this - but I would like to _try_ to use less ENV variables inside of classes in favor of setting them in the constructors or passing them in to the methods. The exception would be the rake task templates that we create. So If you want to use an ENV variable, then grab it in the rake task and pass it in to the method.

Reply:
Yeah, I'm partial to the constructor route myself. This was a one off case in an otherwise established template. I moved this to the task. Maybe not in this case, but deeper utilization may need their own classes in the future.